dali/kernels/signal/window/extract_windows_gpu.cuh

```diff
@@ -237,7 +240,7 @@ __global__ void ExtractVerticalWindowsBatchedKernel(
       pos,                        // first window index
       dst, num_windows, stride,   // output
       src, length,                // input
-      window, out_win_len, in_win_len, win_center, step, reflect,  // windowing options
+      window, out_win_len, in_win_start, in_win_len, win_center, step, reflect,  // win options
```

Comment:
Why have you truncated `windowing`? Let me guess... linter?

Reply:
yes. It seemed better than breaking the line or adding NOLINT